core/setup.py

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env python
-from setuptools import find_namespace_packages
-from setuptools import setup
 import os
+import sys
+
+from setuptools import setup
+try:
+    from setuptools import find_namespace_packages
+except ImportError:
+    # the user has a downlevel version of setuptools.
+    print('Error: dbt requires v40.1.0 or higher of setuptools.')
```

Comment:
I know this is annoying, but:
```suggestion
    print('Error: dbt requires setuptools v40.1.0 or higher.')
```